src/sections/collection-list.liquid

```diff
@@ -1,38 +1,45 @@
-{% if section.settings.title != blank %}
-  <h2>{{ section.settings.title | escape }}</h2>
-{% endif %}
+{%- comment -%}
+  This is a required section for the Shopify Theme Store.
+  It is available when you add "Collection list" section on the Theme Editor.
 
-{% for block in section.blocks limit: section.blocks.size %}
-  <div {{ block.shopify_attributes }}>
+  Theme Store required settings
+  - Collection
+
+  Theme Store optional settings
+  - Heading: title of the collection list section
+{%- endcomment -%}
+<section class="collection-list">
+  {% if section.settings.title != blank %}
+    <h2 class="collection-list__title">{{ section.settings.title | escape }}</h2>
+  {% endif %}
+
+  {% for block in section.blocks limit: section.blocks.size %}
```

Comment:
Adding the `limit` here is redundant, we should be able to remove this since it won't ever loop over any blocks that aren't defined.